src/models/room-state.js

```diff
@@ -663,25 +677,29 @@ RoomState.prototype._maySendEventOfType = function(eventType, userId, state) {
 
     let state_default = 0;
     let events_default = 0;
+
     if (power_levels_event) {
         power_levels = power_levels_event.getContent();
         events_levels = power_levels.events || {};
 
-        if (utils.isNumber(power_levels.state_default)) {
+        if (Number.isFinite(power_levels.state_default)) {
```

Comment:
why the change?

Reply:
Better to use things that are now available in the standard JS as opposed to rolling our own?

Reply:
There is another `utils.isNumber` below we should probably change for consistency